### DISCUSSION TITLE
Auto crafting Logistic pipes with new programmers?

### DISCUSSION THREAD

**Discussion by EShezzerA** [RS485/LogisticsPipes#1632] [Q&A] [answered]
Does anyone know how to auto craft Logistic pipes with the new programmers?
I'm using LP Crafting Tables but when I order say 10 crafting logistic pipes, it says I'm missing 9 programmers.
I have some what a solution at the moment but I can't see it being the correct way of doing it?
but it's to leave the programmer (crafting pipe) in the crafting table, and click on the crafting pipe fitted to the crafting table and just remove the programmer from the crafting recipe

**Comment by bziemons** [comment, marked as answer]
what you suggest is exactly the correct way of doing it :)

**Reply by EShezzerA** [reply]
Thank you, I've been trying to do this for several days and only thought of the (suggestion) after I wrote the Question and it worked.
